src/js/utils/normalize.ts

```diff
@@ -0,0 +1,15 @@
+import { normalize } from '@sentry/utils';
+
+/**
+ * Converts any input into a valid record with string keys.
+ */
+export function convertToNormalizedObject(wat: unknown): Record<string, unknown> {
+  const normalized: unknown = normalize(wat);
+  if (typeof normalized !== 'object') {
+    return {
+      unknown: normalized,
```

Comment:
L: if `unknown` is the new key in case it's not a Key/Value obj, we can use `value` instead similar to Java for `Contexts`.

Reply:
Or `unknown` is a type here?

Reply:
`unknown` is a key here. Sure let's use `value` instead, I would love to keep this consistent.